zebra-state/src/service/finalized_state/zebra_db/shielded.rs

```diff
@@ -243,14 +276,6 @@ impl DiskWriteBatch {
         self.zs_insert(sapling_anchors, sapling_root, ());
         self.zs_insert(orchard_anchors, orchard_root, ());
```

Comment:
It also seems possible to use a height -> anchor -> tree map in the finalized state (which would entail adding the height -> anchor maps to all pools and the anchor -> tree map for Sapling and Orchard). That would trade performance (one additional lookup) for (maybe?) space (no need to store repeated trees). But we can save space without performance loss with https://github.com/ZcashFoundation/zebra/issues/3757

So you don't need to do anything, I'm rambling, but maybe this comment will be useful :rofl: